Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropperThumb.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The <see cref="ImageCropperThumb"/> control is used for <see cref="ImageCropper"/>.
+    /// </summary>
+    public sealed class ImageCropperThumb : Control
+    {
+        private readonly CompositeTransform _layoutTransform = new CompositeTransform();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ImageCropperThumb"/> class.
+        /// </summary>
+        public ImageCropperThumb()
+        {
+            DefaultStyleKey = typeof(ImageCropperThumb);
+            RenderTransform = _layoutTransform;
+            ManipulationMode = ManipulationModes.TranslateX | ManipulationModes.TranslateY;
+            SizeChanged += ImageCropperThumb_SizeChanged;
+        }
+
+        private void ImageCropperThumb_SizeChanged(object sender, SizeChangedEventArgs e)
+        {
+            UpdatePosition();
+        }
+
+        internal PositionTag PositionTag { get; set; }
```

Comment:
I'd probably call the enum `ThumbPosition` and the property `Position`?

Reply:
Yes, `ThumbPosition` is better, I like it.